web-console/src/@core/utils/create-emotion-cache.ts

```diff
@@ -1,5 +0,0 @@
-import createCache from '@emotion/cache'
-
-export const createEmotionCache = () => {
```

Comment:
One thing I notice is that the site (after a hard reload in the browser) is slow again.. this doesn't happen on main.. all navigation stays fast (it's only slow on first/clicks startup) maybe it has to do with removing this?